src/Application/Messenger/Handler/StripePayoutHandler.php

```diff
@@ -117,6 +117,13 @@ public function __invoke(StripePayout $payout): void
             }
 
             if ($donation->getDonationStatus() === DonationStatus::Collected) {
+                // Fix for donations affected by the fee fallback bug where original fees were set low.
+                // We detect this by checking for suspiciously low fees (£0.10 or less) on collected
```

Comment:
may be worth just adding that we know the fee should never be less than £0.20 since as https://biggive.org/our-fees/ we have a fixed fee part of £20p per donation regardless of payment method. Or since this is temp code we can just leave that note here no need to add it to the code.